tour-tip/config.json

```diff
@@ -236,11 +236,11 @@
       "size": "s"
     },
     "content": {
-      "title": "Email briefings subscriptions can now be handled in myFT.",
+      "title": "Email briefings subscriptions can now be handled in myFT. <a href=\"https://www.ft.com/myft/alerts/442fd437-c69e-4697-8de7-db47179c0a7e\" class=\"tour-tip__body-link\" data-trackable=\"email-briefings\"></a>Take a look.",
```

Comment:
Minor tweak: `/442fd437-c69e-4697-8de7-db47179c0a7e` needs to stripped out (that’s JDubz‘s uuid)